packages/composables/src/hooks/useCart/index.ts

```diff
@@ -12,13 +13,18 @@ import { LineItem } from "@shopware-pwa/commons/interfaces/models/checkout/cart/
 import { getApplicationContext } from "@shopware-pwa/composables";
```

Comment:
Regarding the 1. point of support, you asked:

1. import `useNotifications` composable
```import { getApplicationContext, **useNotifications** } from "@shopware-pwa/composables";```

2. get the right function inside a composable:
```
const { pushSuccess, pushError } = useNotifications(root); 
```
use pushSuccess or pushError depending on your needs to show a proper notification. 

for instance:

```
async function submitPromotionCode(promoCode: string) {
try {
    const result = await addPromotionCode(promoCode, apiInstance);
    vuexStore.commit("SET_CART", result);
    pushSuccess(root.$t("promo code added"));

   } catch (error) {
    pushError(error);
   }    
 }
```